doc/src/parts/user_guide.rst

```diff
@@ -514,45 +480,90 @@ The currently set keywords can be obtained with the :py:meth:`~.Plot.get_keyword
+    plot.customize('model_error_band', 'linestyle', '--')  # Set the linestyle for the border of the error band
+    plot.customize('model_error_band', 'linewidth', 3)  # Set the linewidth for the border of the error band
+    plot.customize('model_error_band', 'color', '#DB1F0B')  # Set the color of the error band
+    plot.customize('model_error_band', 'label', 'model error band label')  # Set the label for the error band
+
+    # Analogous to data: set the appearance for the residual
+    plot.customize('residual', 'marker', 'X')
+    plot.customize('residual', 'markersize', 10)
+    plot.customize('residual', 'color', '#600E8F')
+    plot.customize('residual', 'ecolor', '#8F0BDB')
+
+    # Analogous to model error band: set the appearance for the residual error band
+    plot.customize('residual_error_band', 'alpha', 0.2)
+    plot.customize('residual_error_band', 'linestyle', '--')
+    plot.customize('residual_error_band', 'linewidth', 3)
+    plot.customize('residual_error_band', 'color', '#DB1F0B')
+
+To hide entries in the info box you can do this:
+.. code-block::python
+    plot.customize('data', 'label', None)
+
+To hide specific elements of the plot (e.g. error bands) you can do this:
+.. code-block::python
+    plot.customize('model_error_band', 'hide', True)
 
-    # the array length must match the amount of fits handled by this plot object.
-    p.customize('model_error_band', 'hide', [True])
-
-In order to change the name for the data set and suppress the second output, use the following call:
-
+Additionally it is possible to change parameters using matplotlib functions.
+Changing the size of the axis labels is done with the following calls:
 .. code-block:: python
+    import matplotlib as mpl
+    mpl.rc('axes', labelsize=20, titlesize=25)
 
-    p.customize('data', 'label', [(0, "test data"), (1, '__del__')])
-
-Marker type, size and color of the marker and error bars can also be customized:
+Axis Range
+^^^^^^^^^^
 
+The plot range can be set via the :py:attr:`~.Plot.x_range` and :py:attr:`~.Plot.y_range` properties:
 .. code-block:: python
+    plot.x_range = (0.5, 4.5)  # Set the range of the x-axis
+    plot.y_range = (2, 10)  # Set the range of the y-axis
 
-    p.customize('data', 'marker', [(0, 'o'), (1,'o')])
-    p.customize('data', 'markersize', [(0, 5), (1, 5)])
-    p.customize('data', 'color', [(0, 'blue'), (1,'blue')]) # note: although 2nd label is suppressed
-    p.customize('data', 'ecolor', [(0, 'blue'), (1, 'blue')]) # note: although 2nd label is suppressed
+Axis Labels
+^^^^^^^^^^^
 
-The corresponding values for the model function can also be customized:
+By default, the plot will use the labels specified for each dataset (see :ref:'container-labels').
+If multiple fits are plotted to the same figure, the axis labels from the data containers are concatenated while skipping duplicates.
+Alternatively the axis labels can be overwritten for each fit.
```

Comment:
I think this should be either "overridden for each fit" or "set as properties for each fit" depending on what you mean.